data/human/free worlds 0 prologue.txt

```diff
@@ -1320,6 +1322,17 @@ mission "FW Katya 1"
 			
 			`	"Fearsome creatures." She walks away, leaving you even more confused.`
 				decline
+			
+			label gun
+			branch killed
+				random < 1
+			`	You dodge sideways and spin quickly. As you do so you hear a loud bang, and the gun fires, just missing the side of your face.`
+			`	Your assailant turns out to be wearing a Free Worlds uniform and a disappointed expression. "That gun wasn't supposed to be loaded. Who put the ammunition in?" She stalks off, leaving you very confused. It would appear that you've just failed some sort of test.`
```

Comment:
Putting a little more of the original text in:
```suggestion
			`	Your assailant turns out to be wearing a Free Worlds uniform and a disappointed expression. "That gun wasn't supposed to be loaded. What dumb reckless bastard put the ammunition in? That could have splattered brains all over the sidewalk" She stalks off, leaving you very confused. It would appear that you've just failed some sort of test.`
```